lib/stripe/request_signing_authenticator.rb

```diff
@@ -12,13 +12,14 @@ class RequestSigningAuthenticator
 
     attr_reader :auth_token
 
-    def initialize(auth_token)
+    def initialize(auth_token, sign_method)
```

Comment:
is this thing usually called `method` in ruby?

Reply:
no